### PR TITLE
Update stoplight-webinars.yaml

### PR DESCRIPTION
**Pull request by ksatx**
Change webinars wording to include broader video content

### CODE REVIEW

**Comment by ksatx** [comment]
[Change "webinars" to "video content" or "video" on website](https://trello.com/c/H2bEPNUX/255-change-webinars-to-video-content-or-video-on-website)

**Comment by ksatx** [comment]
[Change "webinars" to "video content" or "video" on website](https://trello.com/c/H2bEPNUX/255-change-webinars-to-video-content-or-video-on-website)

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit e8dc377589391b519519c646e62ed935c6288e9e

https://deploy-preview-1263--stoplightio.netlify.com

**Comment by ksatx** [comment]
Fix for issue https://github.com/stoplightio/stoplight.io/issues/1261

**Comment by ksatx** [comment]
Fix for issue https://github.com/stoplightio/stoplight.io/issues/1260

**Comment by ksatx** [comment]
[update list images on video testimonial case studies](https://trello.com/c/jiLrEn0G/281-update-list-images-on-video-testimonial-case-studies)

File: netlify/lists/stoplight-webinars.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 path: /video
 tag: video
-title: Stoplight Webinars
+title: Stoplight API Corner Video Content
```
Comment (rowa97) [inline netlify/lists/stoplight-webinars.yaml]: if you wanted this to match our other content it would be "Video Content | Stoplight API Corner"

**Comment by ksatx** [comment]
@rowa97 I'm doing something wrong here - adding more commits keeps making the checks fail. I'll change that title in my next PR.

**Pull request closed without merging**